recipes/minhook/all/patches/cci.20240114-0001-increase-cmake-min.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index d45414e..533dac6 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -24,7 +24,7 @@
+ #  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ #  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+-cmake_minimum_required(VERSION 3.0)
++cmake_minimum_required(VERSION 3.5)
```

Comment:
```suggestion
+cmake_minimum_required(VERSION 3.15)
```
Might as well make it 3.15, which avoids `shared` and `fPIC` issues and is required by Conan anyway.